### PR TITLE
Fix nested data

### PR DESCRIPTION
**Pull request by julius**

### CODE REVIEW

**Comment by ghost** [comment]
<details>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=10&src=https%3A%2F%2Fgithub.com%2FAdmiralCloud%2Fac-signature">Click on the image</a> for a new way to code review</b>
</summary>

  - Make big changes easier — review code in small groups of related files
  - Know where to start — see the whole change at a glance
  - Take a code tour — explore the change with an interactive tour
  - Make comments and review — all fully sync’ed with github

    [Try it now!](https://app.codesee.io/r/reviews?pr=10&src=https%3A%2F%2Fgithub.com%2FAdmiralCloud%2Fac-signature)
</details>

[Review these changes using an interactive CodeSee Map](https://app.codesee.io/r/reviews?pr=10&src=https%3A%2F%2Fgithub.com%2FAdmiralCloud%2Fac-signature)

#### Legend
CodeSee Map Legend

**Review by markBerlin — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -136,3 +124,24 @@ const acSignature = () => {
...
+    let out = {};
+    Object.keys(obj).sort((a, b) => a.localeCompare(b)).forEach(function (key) {
+      out[key] = deepSort(obj[key]);
+    });
+    return out;
+  }
+
+  return obj;
+
+  function isObject(o) {
```
Comment (MrGutis) [inline index.js:185]: This is interesting solution. I learned something :)
I would have chosen `return o !== null && typeof o === 'object'`

**Review by MrGutis — APPROVED** [review]
@julius I am approving but I had the concerns because of `deepSort` naming. I thought that it would sort not only properties of objects but arrays as well. Because if you send identical data with different order in array - it won't pass the tests. But anyway it's not an issue as API never changes the order in arrays so it should be ok. Please do not name in the feature so generic `deepSort` because it's not general sorting algorithm - it's sorts just objects properties leaving rest in place.

**Comment by julius** [comment]
@MrGutis Good point. I renamed it to deepSortObjectKeys

**Review by markBerlin — APPROVED** [review]

**Pull request merged**